CHANGES.txt

```diff
@@ -5,6 +5,9 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Fixed an issue that caused rewritable subselect to fail if camelcase
+   alias was used.
```

Comment:
I think this description is too impediment specific and can possibly be more generic and meaningful to the user (maybe mentioning that aliases were not case sensitive before and this fixes that)